### DISCUSSION TITLE
gatsby-starter-blog-theme-core default behaviour

### DISCUSSION THREAD

**Discussion by crhayes** [gatsbyjs/gatsby#27717] [Help] [answered]
When I create a new project from `gatsby-starter-blog-theme-core` then `$ gatsby develop` I get the following in the browser:

![image](https://user-images.githubusercontent.com/727536/97615857-32cce180-19f2-11eb-9103-e3897b2aece1.png)

I'm curious why the project isn't rendering the `Posts` and `Post` components that ship with `gatsby-theme-blog-core`?

`gatsby-theme-blog` renders UI by default, but `gatsby-theme-blog-core` does not. If someone can illuminate what they do differently to achieve this I would appreciate it!

**Comment by LekoArts** [comment, marked as answer]
Hi!

What you're seeing is kinda the expected behavior. The `-core` theme describes itself with:

> It includes all of the data structures you need to get up and running building a blog and includes no additional theming or style opinions.

However, the starter you used unfortunately uses an outdated version of the theme (I asked Renovate Bot to open PRs for that now). So once you upgrade the theme to v2 you should see more.

**Reply by crhayes** [reply]
🤦 I didn't think to check the versions... assumed installing the starter would pull the latest `gatsby-theme-blog-core`.

This makes much more sense after seeing the components in the version I have installed:

```jsx
// Post
import React from "react"

export default props => <pre>{JSON.stringify(props.data, null, 2)}</pre>

// Posts
import React from "react"

export default props => <pre>{JSON.stringify(props.data, null, 2)}</pre>
```

Thanks for the reply @LekoArts
